Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/storage/webhdfs/WebHdfsOpParam.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ *
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ * the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.microsoft.azure.hdinsight.sdk.storage.webhdfs;
+
+import com.microsoft.azuretools.azurecommons.helpers.NotNull;
+import org.apache.http.message.BasicNameValuePair;
+
+public class WebHdfsOpParam extends BasicNameValuePair {
+    public static final String NAME = "op";
+    public WebHdfsOpParam(@NotNull String value) {
```

Comment:
You can define static value here for `CREATE` and `OPEN`, such as
```suggestion
    public static final WebHdfsOpParam CREATE = new WebHdfsOpParam("CREATE")
```

And use them directly.